simulation/behavior_tree_plugin/include/behavior_tree_plugin/action_node.hpp

```diff
@@ -88,26 +89,26 @@ class ActionNode : public BT::ActionNodeBase
       BT::InputPort<std::shared_ptr<traffic_simulator::TrafficLightManager>>("traffic_light_manager"),
       BT::InputPort<traffic_simulator::behavior::Request>("request"),
       BT::OutputPort<std::optional<traffic_simulator_msgs::msg::Obstacle>>("obstacle"),
-      BT::OutputPort<std::shared_ptr<traffic_simulator::CanonicalizedEntityStatus>>("updated_status"),
+      BT::OutputPort<std::shared_ptr<traffic_simulator::EntityStatus>>("non_canonicalized_updated_status"),
```

Comment:
Why do you passing uncanonicalized entity status?
I read pull-request description `Canonicalization of EntityStatus in BehaviorTree` section, but I could not understand why we have to change port type.